### PR TITLE
Example/BI Illiteracy

### PR DESCRIPTION
**Pull request by gcattan**
Issue #8 

This is an example of pipeline evaluation with the worst subjects from the Brain Invaders datasets.
The whole dataset took 3 days to run (but in parallel with the simulation for the financial data PR), and eventually failed due to a MOABB bug.

Below you can find an example with the worst subjects of the VR dataset only (6 "subjects").

Figure_1

A few comments:
- `QuantumSVM` is in fact a classical SVM here. I disabled it because it returned 50% systematically. @toncho11 do you have time to look into it when you come back? From memory, we obtained something around 0.75 with bi2012. This is weird.
- I think the voting classifier is not appropriate here. We probably want to find the labels on which quantum and classical MDM differ and train an XGBoost on these specific samples.

We can already discuss this example, and proceed to some adjustments before running the whole dataset.

@qbarthelemy FYI

### CODE REVIEW

File: examples/ERP/classify_P300_bi_illiteracy.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+# ##############################################################################
+# # Plot Results
+# # ----------------
+# #
+# # Here we plot the results to compare two pipelines
+
+fig, ax = plt.subplots(facecolor="white", figsize=[8, 4])
+
+sns.stripplot(
```
[the remark this answers is not in the document]
Comment (gcattan) [reply examples/ERP/classify_P300_bi_illiteracy.py]: Ok. It will look like this but with a dotted line instead (result for the 1 subject in the dataset):

Figure_1

There is still some problem with the QSVC, which we need to investigate, but do you see other pipelines to test?

[the remark this answers is not in the document]
Comment (qbarthelemy) [reply examples/ERP/classify_P300_bi_illiteracy.py]: It's ok for me.

File: examples/ERP/classify_P300_bi_illiteracy.py [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+    QuantumClassifierWithDefaultRiemannianPipeline,
+)
+
+
+print(__doc__)
+
+########################################## Judge classifier
+
+
+# Grossi et al
```
Comment (qbarthelemy) [inline examples/ERP/classify_P300_bi_illiteracy.py:62]: Can you add the reference?

File: examples/ERP/classify_P300_bi_illiteracy.py [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+
+pipelines[PIP.ERPCov_QMDM_Dist.value] = QuantumMDMWithRiemannianPipeline(
+    convex_metric="distance", quantum=True
+)
+placeholder(PIP.ERPCov_QMDM_Dist.value)
+
+pipelines[PIP.ERPCov_QMDM_Dist.value] = QuantumMDMWithRiemannianPipeline(
+    convex_metric="distance", quantum=True
+)
+placeholder(PIP.ERPCov_QMDM_Dist.value)
```
Comment (qbarthelemy) [inline examples/ERP/classify_P300_bi_illiteracy.py:245]: Why is this pipeline called "ERPCov"?
because it does not use [ERPCovariances](https://pyriemann.readthedocs.io/en/latest/generated/pyriemann.estimation.ERPCovariances.html)

Comment (qbarthelemy) [reply examples/ERP/classify_P300_bi_illiteracy.py:245]: Sorry, I hadn't seen the inside of the code QuantumMDMWithRiemannianPipeline.

File: examples/ERP/classify_P300_bi_illiteracy.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gcattan) [reply examples/ERP/classify_P300_bi_illiteracy.py]: Super, thanks :) We just need to solve the issue with QSVC. I see it is working correctly with the loans dataset, and I see I have the same problem if I skip the wrapper, and create the pipeline directly with sklearn. Maybe it is just wrongly parametred.

**Comment by gcattan** [comment]
@toncho11 FYI
I spent some time investigating the bug with QSVM. Here are a few observations:
- The bug also appears when bypassing the `QuantumClassifierWithDefaultRiemannianPipeline` and using directly the QuanticSVM.
- I played with some values of C and gamma without effect
- I have tried to revert from `AerSimulator` to `BasicAer` in `quantum_devices.py`, without effect.

QSVM is still able to generalize somehow with other datasets, and we have the results from NBT. So, I guess it is a matter of finding which parameters need to be adapted.

**Comment by gcattan** [comment]
QSVM accuracy is 100% on the training data. Seems overfitted.

**Review by qbarthelemy — APPROVED** [review]

**Comment by gcattan** [comment]
Thanks for the review @qbarthelemy !
